scripts/ci/build.sh

```diff
@@ -20,10 +20,11 @@ else
     TZ=UTC git show --quiet --date="format-local:%Y.%-m.%-d" --format="${prerelease_source}.%cd" >prerelease.txt
 fi
 
-if [ -n "$CIRCLE_SHA1" ]
-then
-    echo -n "$CIRCLE_SHA1" >commit_hash.txt
-fi
+# We don't want to overwrite the commit for this bug fix
+# if [ -n "$CIRCLE_SHA1" ]
+# then
+#     echo -n "$CIRCLE_SHA1" >commit_hash.txt
+# fi
```

Comment:
This is not necessary now, but, well, it's not like it makes any difference :) Just delete it if you need to rerun for some reason.